### PR TITLE
Only include cache-control: max-age on css and images.

### PR DESCRIPTION
**Pull request by nicksantamaria**
**Issue**:

### What I did

- 
- 

### How to test

- 
- 

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [ ] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

File: packages/nuxt-ripple/nuxt.config.ts [hunk 1]
```diff
@@ -35,10 +35,17 @@ export default defineNuxtConfig({
   },
   nitro: {
     routeRules: {
-      '/_nuxt/**': {
+      // Allow images and css to be cached in the browser (max-age) as long as 
+      // in varnish (s-maxage).
+      '/_nuxt/**/*.{css,png,jpg,svg}': {
```
Comment (nicksantamaria) [inline packages/nuxt-ripple/nuxt.config.ts:40]: File extensions here are indicative only. Need to tune based on what we're actually serving from ripple.

File: packages/nuxt-ripple/nuxt.config.ts [hunk 2]
```diff
@@ -35,10 +35,17 @@ export default defineNuxtConfig({
...
         headers: {
           'cache-control': `public,max-age=${assetCacheTime},s-maxage=${assetCacheTime}`
         }
+      },
+      '/_nuxt/**': {
+        headers: {
+          'cache-control': `public,s-maxage=${assetCacheTime}`
```
Comment (dylankelly) [inline packages/nuxt-ripple/nuxt.config.ts:47]: so should this be shorter then @nicksantamaria ?

Comment (nicksantamaria) [reply packages/nuxt-ripple/nuxt.config.ts:47]: `s-maxage` is the varnish TTL which will respect cache purges. IMHO its fine to have this nice and long.

**Comment by nicksantamaria** [comment]
I think we need to flip the order of these items.

**Pull request closed without merging**